File: templates/templates/EpoxyHello.Avalonia/EpoxyHello.UI/Views/MainWindow.axaml.cs
////////////////////////////////////////////////////////////////////////////
//
// Epoxy template source code.
// Write your own copyright and note.
// (You can use https://github.com/rubicon-oss/LicenseHeaderManager)
//
////////////////////////////////////////////////////////////////////////////

using Avalonia;
using Avalonia.Controls;
using Avalonia.Markup.Xaml;

namespace EpoxyHello.Views
{
    public sealed class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
#if DEBUG
            this.AttachDevTools();
#endif
        }

        private void InitializeComponent() =>
            AvaloniaXamlLoader.Load(this);
    }
}

File: templates/templates/FSharp.EpoxyHello.Avalonia11/EpoxyHello.UI/Views/MainWindow.axaml.fs
namespace EpoxyHello.Views

open Avalonia.Controls
open Avalonia.Markup.Xaml

[<Sealed>]
type MainWindow() as self =
    inherit Window()

    do
        self.InitializeComponent()

    member private self.InitializeComponent() =
        AvaloniaXamlLoader.Load(self)
